README.md

```diff
@@ -149,6 +149,19 @@ because there's really no magic to do that for you... :)
 }
 ```
 
+## `trello-fields.json`
```

Comment:
TODO: for the future. Let's make a little script to go in `./utils/` that plucks all of the customField names and IDs.

And maybe another for labels or something like that.

Reply:
sounds like a small project I could handle...I also need to add that to the readme a couple lines up in the quickstart section